cmd/influx/setup.go

```diff
@@ -141,17 +141,20 @@ func setupF(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("instance at %q has already been setup", activeConfig.Host)
 	}
 
-	existingConfigs := make(config.Configs)
 	if _, err := os.Stat(dPath); err == nil {
-		existingConfigs, _ = localConfigSVC.ListConfigs()
-		// ignore the error if found nothing
-		if setupFlags.name == "" {
-			return errors.New("flag name is required if you already have existing configs")
-		}
-		if _, ok := existingConfigs[setupFlags.name]; ok {
-			return &influxdb.Error{
-				Code: influxdb.EConflict,
-				Msg:  fmt.Sprintf("config name %q already existed", setupFlags.name),
+		existingConfigs, _ := localConfigSVC.ListConfigs()
+		if len(existingConfigs) > 0 {
```

Comment:
can we invert this logic to save an indent level?

for example, could this be:

```
if len(existingConfigs) == 0 {
 break
}
```

Reply:
`break` doesn't seem to work, since it's not in a loop. I'll poke at it some more.

Reply:
Solution: move this logic to a separate function.